full_coverage/DebugCell[] coarse_cells
full_coverage/DebugCell[] robot_cells
string frame_id
